app/views/posts/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Here is a post by a current user</h1>
\ No newline at end of file
+<div>
+    <div class='flex justify-between gap-10'>
+        <h1><%= @post.title%></h1>
+        <div class='flex self-end gap-1'>
+            <p>Comments:<%= @post.comments_counter %>, </p>
+            <p>Likes: <%= @post.likes_counter%></p>  
+        </div>
+    </div>
+    <%= @post.text%></p>
+    <% @post.comments.each do |comment|%>
+      <p><%= comment.author.name %>: <%= comment.text%></p>
+    <%end%>
+    
```

Comment:
- [ ]  Do the same for your Post show page.

Your Implementation | Original Design
:---:|:---:
![image](https://github.com/biruk-bereka/Blogify/assets/79058364/a6681588-94a8-4332-b1e7-3c71d744799e) |![image](https://github.com/biruk-bereka/Blogify/assets/79058364/6266b31f-3d27-4613-ae20-9f11d2cf2407)